ext/tidy/tests/019.phpt

```diff
@@ -28,13 +28,13 @@ tidy_repair_file($l, $l, $l ,$l); // This doesn't emit any warning, TODO look in
 echo "Done\n";
 ?>
 --EXPECTF--
-Warning: tidy_repair_string(): Could not load configuration file "1" in %s on line %d
+Warning: tidy_repair_string(): Could not load the Tidy configuration file "1" in %s on line %d
```

Comment:
```suggestion
Warning: tidy_repair_string(): Could not load Tidy configuration file "1" in %s on line %d
```

Reply:
One of the old messages used "the" for the configuration, this one previously not. That's why I changed it for consistency